==3848== Memcheck, a memory error detector
==3848== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==3848== Using Valgrind-3.16.1 and LibVEX; rerun with -h for copyright info
==3848== Command: ./cub3D own_test.cub
==3848== 
==3848== 
==3848== HEAP SUMMARY:
==3848==     in use at exit: 46,935,829 bytes in 2,646 blocks
==3848==   total heap usage: 69,088 allocs, 66,442 frees, 261,383,508 bytes allocated
==3848== 
==3848== 24 bytes in 1 blocks are definitely lost in loss record 62 of 2,010
==3848==    at 0x483877F: malloc (vg_replace_malloc.c:307)
==3848==    by 0x113671: mlx_load_png (in /home/pythongermany/vboxshare/42projects/42_cub3d/cub3D)
==3848==    by 0x10BAB3: init_mlx (in /home/pythongermany/vboxshare/42projects/42_cub3d/cub3D)
==3848==    by 0x10BC71: init_data (in /home/pythongermany/vboxshare/42projects/42_cub3d/cub3D)
==3848==    by 0x10A62B: main (in /home/pythongermany/vboxshare/42projects/42_cub3d/cub3D)
==3848== 
==3848== 64 bytes in 1 blocks are definitely lost in loss record 1,850 of 2,010
==3848==    at 0x4838DEF: operator new(unsigned long) (vg_replace_malloc.c:342)
==3848==    by 0x6B2E879: ???
==3848==    by 0x63F37DF: ???
==3848==    by 0x400FFE1: call_init.part.0 (dl-init.c:72)
==3848==    by 0x40100E8: call_init (dl-init.c:30)
==3848==    by 0x40100E8: _dl_init (dl-init.c:119)
==3848==    by 0x4B67AEC: _dl_catch_exception (dl-error-skeleton.c:182)
==3848==    by 0x4014363: dl_open_worker (dl-open.c:758)
==3848==    by 0x4B67A8F: _dl_catch_exception (dl-error-skeleton.c:208)
==3848==    by 0x40138F9: _dl_open (dl-open.c:837)
==3848==    by 0x4A0A257: dlopen_doit (dlopen.c:66)
==3848==    by 0x4B67A8F: _dl_catch_exception (dl-error-skeleton.c:208)
==3848==    by 0x4B67B4E: _dl_catch_error (dl-error-skeleton.c:227)
==3848== 
==3848== 5,355 bytes in 255 blocks are definitely lost in loss record 2,003 of 2,010
==3848==    at 0x483877F: malloc (vg_replace_malloc.c:307)
==3848==    by 0x4ABBF3A: strdup (strdup.c:42)
==3848==    by 0x60144A6: ???
==3848==    by 0x6014254: ???
==3848==    by 0x5FB7F73: ???
==3848==    by 0x5EDB436: ???
==3848==    by 0x5EDBCE8: ???
==3848==    by 0x5F570F0: ???
==3848==    by 0x5F58417: ???
==3848==    by 0x4876D02: ??? (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==3848==    by 0x4866477: ??? (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==3848==    by 0x486BB6A: glfwCreateWindow (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==3848== 
==3848== 160,000 bytes in 1 blocks are possibly lost in loss record 2,008 of 2,010
==3848==    at 0x483AB65: calloc (vg_replace_malloc.c:760)
==3848==    by 0x11A9CE: decodeGeneric (in /home/pythongermany/vboxshare/42projects/42_cub3d/cub3D)
==3848==    by 0x11F559: lodepng_decode_memory (in /home/pythongermany/vboxshare/42projects/42_cub3d/cub3D)
==3848==    by 0x11FEA3: lodepng_decode32_file (in /home/pythongermany/vboxshare/42projects/42_cub3d/cub3D)
==3848==    by 0x113690: mlx_load_png (in /home/pythongermany/vboxshare/42projects/42_cub3d/cub3D)
==3848==    by 0x10BAB3: init_mlx (in /home/pythongermany/vboxshare/42projects/42_cub3d/cub3D)
==3848==    by 0x10BC71: init_data (in /home/pythongermany/vboxshare/42projects/42_cub3d/cub3D)
==3848==    by 0x10A62B: main (in /home/pythongermany/vboxshare/42projects/42_cub3d/cub3D)
==3848== 
==3848== LEAK SUMMARY:
==3848==    definitely lost: 5,443 bytes in 257 blocks
==3848==    indirectly lost: 0 bytes in 0 blocks
==3848==      possibly lost: 160,000 bytes in 1 blocks
==3848==    still reachable: 46,770,354 bytes in 2,387 blocks
==3848==         suppressed: 32 bytes in 1 blocks
==3848== Reachable blocks (those to which a pointer was found) are not shown.
==3848== To see them, rerun with: --leak-check=full --show-leak-kinds=all
==3848== 
==3848== For lists of detected and suppressed errors, rerun with: -s
==3848== ERROR SUMMARY: 4 errors from 4 contexts (suppressed: 0 from 0)
